README.md

```diff
@@ -40,6 +40,23 @@ $ python manage.py runserver
 ```
 
 
+## Installation - docker with nginx and gunicorn
+
+Build the image using docker
+
+```bash
+$ docker build -t pkpdapp .
+```
+
+Run the server
+
+```bash
+$ docker run -it -p 8020:8020 -e DJANGO_SUPERUSER_USERNAME=admin -e 
```

Comment:
Could we change this to 8020:8000 just to have the same default address as Django. And could we also add `\` for the breaks, so people can just copy paste the command and run the server.